content/post/project-dynamic-observation-and-interruption-of-sparql-queries.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "Dynamic Observation and Interruption of SPARQL Queries"
+date: 2024-03-29T03:19:19+01:00
+author: "Robin Textor-Falconi"
+authorAvatar: "img/ada.jpg"
+tags: ["SPARQL", "WebSocket", "HTTP", "Synchronization", "Atomics", "QLever"]
+categories: ["project"]
+image: "img/project-dynamic-observation-and-interruption-of-sparql-queries/teaser.png"
+draft: true
```

Comment:
```suggestion
draft: false
```
(Then we don't wonder why it is not shown when deploying the post)

Reply:
I probably should adjust the date too before merging this so it matches the moment it gets published